### PR TITLE
[ADD] User registration and login using Django

### PR DESCRIPTION
**Pull request by spaul-12**
Completed the backend code for authentication and connected it with the login page.
issue#117

Screenshot from 2023-06-09 11-14-47
Screenshot from 2023-06-09 11-14-11
Screenshot from 2023-06-09 11-25-33
Screenshot from 2023-06-09 11-26-12

### CODE REVIEW

**Comment by spaul-12** [comment]
@JigarJoshi04 @codervivek5 Please review this PR and merge it.

File: Backend Code/VigyBag/VigyBag/__pycache__/__init__.cpython-310.pyc
Comment (JigarJoshi04) [inline Backend Code/VigyBag/VigyBag/__pycache__/__init__.cpython-310.pyc:1]: Do not push `*.pyc` files

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+env/
```
Comment (JigarJoshi04) [inline .gitignore:1]:
```suggestion
env/
*.pyc
```

File: Backend Code/VigyBag/VigyBag/settings.py [hunk 2]
```diff
@@ -123,3 +176,9 @@
 # https://docs.djangoproject.com/en/3.2/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+#CORS_ALLOW_ALL_ORIGINS: True
+CORS_ALLOWED_ORIGINS = [
+    "http://localhost:5500",
+
```
Comment (JigarJoshi04) [inline Backend Code/VigyBag/VigyBag/settings.py:183]: I guess,  this is for your local setup.

**Comment by JigarJoshi04** [comment]
Why have we choosen a route to start from very base of the user?
We should use the django's user model and make changes in it.
https://docs.djangoproject.com/en/4.2/topics/auth/default/

**Review by JigarJoshi04 — CHANGES_REQUESTED** [review]

File: Backend Code/VigyBag/VigyBag/settings.py (hunk 2 above)
Comment (spaul-12) [reply Backend Code/VigyBag/VigyBag/settings.py:183]: Yes, we can add other origins as well when required

**Comment by spaul-12** [comment]
@JigarJoshi04 I have made the  changes.

**Pull request merged**